pkg/kubectl/cmd/taint.go

```diff
@@ -40,6 +40,13 @@ import (
 	utiltaints "k8s.io/kubernetes/pkg/util/taints"
 )
 
+const (
+	MODIFIED    = "modified"
```

Comment:
@timothysc These are the constants that are being used in output of kubectl, doesn't it make sense to have them here instead of API?
@jayunit100 This gets appended to 'node 'XYZ' from printSuccess at https://github.com/kubernetes/kubernetes/pull/43171/files#diff-92dd749c11309adffd4b2b2112b36303R364. I guess a better way would be to remove 'node 'XYZ' ' part when printing and tell if the taint was added or deleted. I guess that will be a different PR but if you want it, I can include it here.